src/app/components/Results/Results.jsx

```diff
@@ -48,17 +48,17 @@ class Results extends React.Component {
 
   getResultsSort() {
     return sortingOpts.map((d, i) => {
-      return (<li role="region" key={i} onClick={e => this.sortResultsBy(e, d.val, d.label)}>
-        {d.label}
+      return (<li role="region" key={i}>
+        <a href="#" onClick={e => this.sortResultsBy(e, d.val, d.label)}>{d.label}</a>
```

Comment:
I see this weird flash thing where the `#` briefly appears in the url before updating to the query. In `sortResultsBy`, make the first line of the function `e.preventDefault();`. That will prevent the default `a` element action of adding `#` to the url.